designs/helm-chart-autogenerate-plugin.md

```diff
@@ -0,0 +1,213 @@
+| Authors       | Creation Date | Status      | Extra |
+|---------------|---------------|-------------|---|
+| @name | date | Implementeble | - |
+
+# Auto-generate Helm Chart Plugin
```

Comment:
```suggestion
# Allow users package Operator with HelmCharts
```

I think our goal is to allow to distribute the Operator done with Kubebuilder via HelmCharts.
We might be able to achive the goal using the Kubebuilder Plugin ecosystem. See: https://book.kubebuilder.io/plugins/plugins

Reply:
+1 for what @camilamacedo86 said, but I believe "Helm Charts" (separated by space) is the common way to write it.